Identity.API/Controllers/AuthenticationController.cs

```diff
@@ -1,8 +1,35 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Identity.BusinessLogic.DTOs;
+using Identity.BusinessLogic.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
 
 namespace Identity.API.Controllers
 {
-    public class AuthenticationController : ControllerBase
+    [Route("api/authentication")]
+    [ApiController]
+    public class AuthenticationController(IAuthenticationService authenticationService) : ControllerBase
     {
+        private readonly IAuthenticationService _authenticationService = authenticationService;
+
+        [HttpPost]
+        [Route("login")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> LoginAsync([FromBody] LoginDTO login)
+        {
```

Comment:
it is necessary to pass CancellationToken to controller methods and further to services